internal/models/ekscluster/vpccni_addon_config.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package models
+
+import (
+	"github.com/go-openapi/swag"
+)
+
+// VmwareTanzuManageV1alpha1EksclusterVpcCniAddonConfig EKS vpc-cni addon configuration
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.ekscluster.VpcCniAddonConfig
+type VmwareTanzuManageV1alpha1EksclusterVpcCniAddonConfig struct {
+
+	// Enable the Kubernetes vpc-cni addon.
```

Comment:
VP CNI is enabled by default, this comment should be fixed

Reply:
@DheerajSShetty  fixed this.